src/Components/UserSignUp.js

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from 'react';
+import axios from 'axios';
+import { Link, useNavigate } from 'react-router-dom';
+import '../Sign-Up.css';
+
+function UserSignUp() {
```

Comment:
- Please, consider using arrow funcs according to Javascript ES6 good practices.